### DISCUSSION TITLE
Keep drop down open onChange

### DISCUSSION THREAD

**Discussion by loriemoffitt** [primefaces/community#2672] [PrimeReact]
How to get the multiselect component to keep the dropdown panel open so the user can make multiple selections?   Is it possible to access the "show" method inside the onChange method to get the panel to stay open? I've tried to preventDefault and stopPropigation and a ton of other things and nothing is working...

**Comment by workstuff-arch** [comment]
New here, looking to do the exact same thing and I've been looking for ways to call methods shown in the documentation, hoping a call to show would work. Any luck?
